server.js

```diff
@@ -1,30 +1,58 @@
-// Log memory usage before doing GC. Seems to blow up the file limit sooner than without.
-var gcLog = function () {
-  console.log(new Date().toISOString() + JSON.stringify(process.memoryUsage()));
-  gc();
-};
+var fs = require('fs');
+var ws = require('ws');
+var http = require('http');
+var randomBytes = require('crypto').randomBytes;
```

Comment:
Hey, I get it you like having lots of variables laying around. I kid, but I have to admit this way is more idiomatic.